examples/fedopt_example/metrics.py

```diff
@@ -75,54 +75,69 @@ def compute_metrics(self) -> Metrics:
         return metrics
 
 
-class ClientMetrics:
+class CustomMetricMeter(MetricMeter):
```

Comment:
Defining our own MetricMeter, as we're going to use a custom class to accrue metric information.

Reply:
Just for future reference, in #69 I based of this branch and made some slight updates to the code in this file. First, by having metrics store their own state, we eliminate the need for having MetricMeters. I updated this to be a "Compound Metric" which stores information that is relevant to compute one or more related metrics. All in all, it does not change very much though.